app/templates/gentelella/admin/base.html

```diff
@@ -5,7 +5,7 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>{% block title %}{% endblock %} - {{ app_name }}</title>
+    <title>{{ app_name }} - {% block title %}{% endblock %}</title>
```

Comment:
With this change, all pages will come like "Open Event - Login", "Open Event - Manage events", "{app name} - {page name}" etc.. Is this right? 
I think "{page name} - {app name}" is more natural and practical. 
Only on homepage we should have {app name} - {tagline}.   cc @mariobehling